BazelExtensions/xchammerconfig.bzl

```diff
@@ -56,13 +56,22 @@ def target_config(
 def project_config(
         paths,  # [String]?
         build_bazel_platform_options = None,  # [String: [String]]?
+        bazel_build_service_config = None, # [String: XCHammerTargetConfig]?
```

Comment:
The data type here is `BazelBuildServiceConfig`?

Reply:
Yes, check that type in `Sources/XCHammer/XCHammerConfig.swift`